documentation/docs/02-runes/03-$derived.md

```diff
@@ -52,6 +52,37 @@ Anything read synchronously inside the `$derived` expression (or `$derived.by` f
 
 To exempt a piece of state from being treated as a dependency, use [`untrack`](svelte#untrack).
 
+## Overriding derived values
+
+Derived expressions are recalculated when their dependencies change, but you can temporarily override their values by reassigning them. This can be useful for things like _optimistic UI_, where a value is derived from the 'source of truth' (such as data from your server) but you'd like to show immediate feedback to the user:
```

Comment:
I wonder if we should mention that you can use `const` if you want to prevent anyone from overriding the value

Reply:
done